src/main/java/ToDoApplication/DAO/BaseDao.java

```diff
@@ -0,0 +1,52 @@
+package ToDoApplication.DAO;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+abstract class BaseDao {
+
+
+    protected void closeResource(Connection connection, PreparedStatement preparedStatement) {
+        if (connection != null) {
```

Comment:
@Greddyy, you have two same try-catch statements in one methods. It would be better to wrap up all lines in method with single try-catch statement